### PR TITLE
MARBLE-1048 clean up allMetadata to show relevant results

### PR DESCRIPTION
**Pull request by jmille79**

### CODE REVIEW

File: scripts/gatsby-source-iiif/indexSearch.js [hunk 1]
```diff
@@ -102,18 +102,18 @@ const getSearchDataFromManifest = (manifest) => {
...
-      search['allMetadata'] += ' ' + manifest[key]
+      search.allMetadata.push(manifest[key])
     }
   })
-  search['allMetadata'] += ' ' + loadSubItemTitles(manifest)
-  search['allMetadata'] += ' ' + search.repository
-  search['allMetadata'] += ' ' + search.centuryTag.join(' ')
-  search['allMetadata'] += ' ' + search.themeTag.join(' ')
-
+  search.allMetadata.push(loadSubItemTitles(manifest))
```
Comment (jonhartzler) [inline scripts/gatsby-source-iiif/indexSearch.js:112]: how does changing the data type to an array change how the highlights come back?

File: @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { jsx } from 'theme-ui'
 
 const HitDisplay = ({ hits, defaultDisplay }) => {
   const referal = { type: 'search', query: window.location.search }
+  console.log(hits)
```
Comment (danthewolfe) [inline @ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/index.js:12]: Remove `console.log`.

File: scripts/gatsby-source-iiif/indexSearch.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ const loadSubItemTitles = (manifest) => {
```
Comment (danthewolfe) [inline scripts/gatsby-source-iiif/indexSearch.js:69]: I'm a little wary of magic strings, but I think there is a case for that here. Can you at least define it as a variable at top and reference that? @jonhartzler is this going to affect indexing?
```
const stringSplit = '::'
```

Comment (jonhartzler) [reply scripts/gatsby-source-iiif/indexSearch.js:69]: yeah.   I think that this is fine for now.  I think that we really are going to still have to work on this section in the future.  I suspect the real problem is that we will probably need to not do this simple blob field index and if we do we may have to do some other system for the highlighting or I guess we could format the blob with HTML and display it.  anyway  I think it moves this forward.

**Pull request merged**